CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 
 - The `Constructor` and `IsVariant` derives now generate `const fn` functions.
+- Static methods derived by `IsVariant` are now marked `#[must_use]`.
```

Comment:
It would be nice to also add the pull request reference, as is done below for some changes.